src/api/app/controllers/webui/projects/rebuild_times_controller.rb

```diff
@@ -56,12 +55,8 @@ def call_diststats(bdep, jobs)
         @rebuildtime = 0
 
         indir = Dir.mktmpdir
-        f = File.open(indir + '/_builddepinfo.xml', 'w')
```

Comment:
please give this a readable name such as `file`

Reply:
no, I won't. f is a perfectly readable variable for a file that is used in one line

Reply:
I don't think it is readable :unamused:

Reply:
The 3 extra letters won't hurt.

Reply:
Fixing actual bugs won't hurt - annoying co-developers with discussions about variable names *does* hurt

Reply:
It's about code style.

Reply:
"We are using Rubocop to check the code style. We are following the Rubocop defaults, with the exceptions in .rubocop.yml. We are not following any other style conventions." 

Code style my ...